crud.py

```diff
@@ -0,0 +1,46 @@
+from sqlalchemy.orm import Session
+
+from db import models
+import schemas
+
+
+def get_all_books(db: Session, skip: int = 0, limit: int = 5):
+    return db.query(models.Book).offset(skip).limit(limit).all()
+
+
+def create_book(db: Session, book: schemas.BookCreate):
+    db_book = models.Book(
+        title=book.title,
+        summary=book.summary,
+        publication_date=book.publication_date,
+        author_id=book.author_id
+    )
+    db.add(db_book)
+    db.commit()
+    db.refresh(db_book)
+
+    return db_book
+
+
+def get_books_by_author_id(db: Session, author_id: int):
+    return db.query(models.Book).filter(models.Book.author_id == author_id)
```

Comment:
The `get_books_by_author_id` function returns a query object instead of a list of books. To return the actual list of books, you should call `.all()` at the end of the query.